firewall.tendrl.yml

```diff
@@ -12,7 +12,7 @@
         name: 'firewalld'
         state: present
       register: task_result
-      until: task_result|success
+      until: task_result is success
```

Comment:
I think, that `irewall.tendrl.yml` file is not part of this repository anymore, so you don't need to change this file to avoid conflict during rebase.

Reply:
Yep, I will resolve this during merge.